api/external/applications/applications.proto

```diff
@@ -278,36 +487,44 @@ message HealthCounts {
 //
 // Example: core/redis/0.1.0/8743278934278923
 message PackageIdent {
+  // Chef Habitat origin.
   string origin = 1;
+  // Chef Habitat package name.
   string name = 2;
+  // Chef Habitat release version number.
   string version = 3;
+  // Chef Habitat release buildstamp.
   string release = 4;
 }
 
+// List of service groups.
 message ServiceGroups  {
+  // List of service groups.
   repeated ServiceGroup service_groups = 1;
 }
 
 // The ServiceStatus enum describes the status of the service
-// @afiune have we defined these states somewhere?
+// Currently unimplemented
 enum ServiceStatus {
   RUNNING = 0;
   INITIALIZING = 1;
   DEPLOYING = 2;
   DOWN = 3;
 }
 
-// The HealthStatus enum matches the habitat implementation for health-check status:
+// The HealthStatus enumerable matches the Chef Habitat implementation for health-check status:
 // => https://www.habitat.sh/docs/reference/#health-check
+// For a health status within a service group.
+// *critical* means that one or more services are in critical condition.
+// *warning* means that one or more services have a warning, but none are in critical condition.
+// *unknown* means that one or more services have not responded, but all of the remaining nodes responded to the health check as "OK".
+// *OK* means that all of the services are OK and all have responded to the health check.
+// *none* means that there is no health check information.
```

Comment:
This looks off in the UI.

![Screen Shot 2019-12-10 at 4 58 19 PM](https://user-images.githubusercontent.com/22962/70582261-b9749800-1b6e-11ea-910f-800500b9d282.png)

I'd suggest applying something like this completely untested markdown formatting:

```
// The HealthStatus enumerable matches [the Chef Habitat implementation for health-check status](https://www.habitat.sh/docs/reference/#health-check).
// For a health status within a service group.
// * *critical* means that one or more services are in critical condition.
// * *warning* means that one or more services have a warning, but none are in critical condition.
// * *unknown* means that one or more services have not responded, but all of the remaining nodes responded to the health check as "OK".
// * *OK* means that all of the services are OK and all have responded to the health check.
// * *none* means that there is no health check information.
```

Reply:
I tried that, but now that I see it I'm wondering if a line space after the sentence & indenting might work.